object_tracker/object_tracker.go

```diff
@@ -66,6 +69,10 @@ func newTracker(ctx context.Context, deps resource.Dependencies, conf resource.C
 		logger:       logger,
 		classCounter: make(map[string]int),
 		tracks:       make(map[string][]objdet.Detection),
+		properties: vision.Properties{ClassificationSupported: false,
```

Comment:
small optional: put ClassificationsSupported on a new line so it's easier to read

Reply:
done